### ISSUE TITLE
How to customize novnc's token?

### ISSUE THREAD

**Issue by jay763190097** [retspen/webvirtmgr#710]
Currently, it's too easy to show Server disconnected (code: 1006)!

**Comment by jay763190097** [comment]
It'd be better if we could set it to expire in 10 mins.

**Comment by jay763190097** [comment]
![image](https://user-images.githubusercontent.com/16317178/216803441-3f694b58-d7db-4b95-b275-c9c98b55fa94.png)

**Comment by catborise** [comment]
i do not think it is related with expiration time. normally it would stays open if instance console accessible. 
might there be an another problem like firewall, connection glitches, unstable instance?

**Comment by jay763190097** [comment]
@catborise any idea to fix it? I use this docker image  primiano/docker-webvirtmgr:latest
